twine/package.py

```diff
@@ -65,6 +65,28 @@ def _safe_name(name: str) -> str:
     return re.sub("[^A-Za-z0-9.]+", "-", name)
 
 
+def _dedent(string: str) -> str:
+    """Remove line continuation suffix used for the ``Description`` metadata field.
+
+    The metadata standard prescribes that continuation lines should be
+    prefixed with 7 spaces followed by a pipe character, however, setuptools
+    used to prefix continuation lines with 8 spaces. Handle both here.
+
+    If not all lines following the first start with either of the accepted
+    prefixes, the string is returned without modifications.
+    """
+    lines = string.splitlines()
+    if (
+        False  # This is here only to force black into something sensible.
```

Comment:
I know exactly why this is here (https://github.com/psf/black/issues/2156). I decided to use `# fmt: skip` to avoid the need to write the comment.